contracts/eos/Token/Token.hpp

```diff
@@ -27,6 +27,7 @@ CONTRACT Token : public contract {
         ACTION retire(asset quantity, string memo);
 
         ACTION transfer(name from, name to, asset quantity, string memo);
+        ACTION transferbyid(name from, name to, uint64_t amount_id, name contract, string memo);
```

Comment:
replace the order of the amount args (contract first) and rename contract to maybe amount_account (??-  not sure we can use more than 12 chars) so that the user will know that the account and id are related.